var/spack/repos/builtin/packages/libflame/package.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libflame(AutotoolsPackage):
+    """libflame is a portable library for dense matrix computations,
+    providing much of the functionality present in LAPACK, developed
+    by current and former members of the Science of High-Performance
+    Computing (SHPC) group in the Institute for Computational
+    Engineering and Sciences at The University of Texas at Austin.
+    libflame includes a compatibility layer, lapack2flame, which
+    includes a complete LAPACK implementation."""
+
+    homepage = "https://www.cs.utexas.edu/~flame/web/libFLAME.html"
+    url      = "https://github.com/flame/libflame/archive/5.1.0.tar.gz"
+    git      = "https://github.com/flame/libflame.git"
+
+    version('develop', branch='master')
+    version('5.1.0', sha256='e7189b750890bd781fe773f366b374518dd1d89a6513d3d6261bf549826384d1')
+
+    provides('lapack', when='+lapack2flame')
+
+    variant('lapack2flame', default=True,
+            description='Map legacy LAPACK routine invocations
+            to their corresponding native C implementations
+            in libflame.')
```

Comment:
This will result in extra spaces in `spack info`. You'll have to either put the entire variant on one line, or wrap it with more quotes.